module/scripts/rolls/extendedRoll.js

```diff
@@ -10,6 +10,7 @@ export async function extendedRoll(rollFormula, messageData, config, flags = [])
 
     //crit/fumble calculation
     if (config.showCrit && (isCrit(roll) || isFumble(roll))) {
+        await game.dice3d?.showForRoll(roll, game.user, true);
```

Comment:
So this roll will only show if there is a crit or fumble?

Reply:
the roll will happen normally if it is not a critical or fumble.

Reply:
Yes, and you only show the roll with 3d dice when it is a crit or fumble. Read the line above